docs/reference/metadata.md

```diff
@@ -170,42 +170,102 @@ while others can only be accessed via the [API][].
 
 ### Indentation Options
 
-Indentation options control aspects of the auto-indentation mechanism.
+Indentation options control aspects of the auto-indentation mechanism;  
+Use regex to match a single line, and make change to the indent-level of the lines following the matched line;  
+The `decreaseIndentPattern` option also change the indent-level of the matched line itself.  
 
 `increaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be indented one level further.
+  lines after current line will be indented one level further.
 
-  ```xml
-  <key>increaseIndentPattern</key>
-  <string>insert regex here</string>
+*format*  
+  ```xml  
+  <key>increaseIndentPattern</key>  
+  <string>insert regex here</string>  
+  ```
+*example*  
+  if in *.tmPreferences*:  
+  ```xml  
+  <key>increaseIndentPattern</key>  
+  <string><![CDATA[^\s*increase$]]></string>  
   ```
+  then in **view**:  
+```
```

Comment:
Please indent this so it's part of the definition block. I also think we can get rid of a couple eample lins below.